AlembicImporter/Packages/com.unity.formats.alembic/Editor/Timeline/AlembicRecorderClipEditor.cs

```diff
@@ -1,12 +1,13 @@
 using System;
 using System.IO;
 using UnityEngine;
+using UnityEngine.Formats.Alembic.Util;
 using UnityEngine.SceneManagement;
 using UnityEditor;
 using UnityEditor.SceneManagement;
 using UnityEngine.Timeline;
 
-namespace UTJ.Alembic
+namespace UnityEditor.Formats.Alembic.Importer
 {
```

Comment:
Do you think we should create a "Timeline" namespace for the timeline bits? I am not too familiar with the package yet but it seems like this is a separate set of features.

Reply:
yes I think that would make more sense than having the timeline stuff under "importer"